the-devastating-chickens/BangazonSite
# Rw search local drop

# Description

A user may filter their searches by location, product name or BOTH. A drop down allows them to select what they search by.



Fixes Issue # https://github.com/the-devastating-chickens/BangazonSite/issues/14

## Type of change

Please delete options that are not relevant.


# Testing Instructions

`git checkout master`
`git fetch --all`
`git checkout rw-SearchLocalDrop`
`Start BangazonSite.sln`
Check ProductsController for comments.
Delete database
Update-Database
Run App
Search 'ill' under all. Do items with 'ill' in the title and city appear? COOL.
Search 'ill' under city. Do items with 'ill' in the city appear? COOL.
Search 'ill' under product. Do items with 'ill' in the product title appear? COOL.



# Checklist:
